runtime/integration-tests/src/generic/envs/runtime_env.rs

```diff
@@ -93,7 +93,7 @@ impl<T: Runtime> Env<T> for RuntimeEnv<T> {
 		call: impl Into<T::RuntimeCallExt>,
 	) -> Result<Balance, DispatchError> {
 		let call: T::RuntimeCallExt = call.into();
-		let info = call.get_dispatch_info();
+		let info = self.parachain_state(|| call.get_dispatch_info());
```

Comment:
NIT. Maybe can be moved directly inside line `126`

Reply:
Seems like `info` is used only in line `126` under an already `parachain_state()`. So maybe `call.get_dispatch_info()` can be moved there. But a super NIT thing.